src/NUnitFramework/nunitlite/OutputManager.cs

```diff
@@ -120,7 +120,7 @@ public void WriteTestFile(ITest test, OutputSpecification spec)
                         "spec");
             }
 
-            outputWriter.WriteTestFile(test, outputPath);
+            outputWriter.WriteTestFile(test, filter, outputPath);
             Console.WriteLine("Tests ({0}) saved as {1}", spec.Format, outputPath);
         }
     }
```

Comment:
My intention was to filter the tests at the lowest possible point to allow any other layer to fiddle with the filter if required.

On further consideration, I can see that it complicates the implementation.

I'll see about the pre-filtered selection of tests.